core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -4247,6 +4247,14 @@ private void checkNullOperand(SqlTester tester, String op) {
     tester.checkNull("CHARACTER_LENGTH(cast(null as varchar(1)))");
   }
 
+  @Test public void testAsciiFunc() {
+    tester.setFor(SqlStdOperatorTable.ASCII);
+    tester.checkScalarExact("ASCII('')", "0");
```

Comment:
Pls also add a test with single char: 'a' or '1'

Reply:
Also what happens when there is a non-ASCII character ?

Reply:
Ok, I will add more test cases.